### ISSUE TITLE
Dram database download issue looking for files with previous datestamps 

### ISSUE THREAD

**Issue by jayramr** [WrightonLabCSU/DRAM#216]
Dear Team,

We are facing error with the dram database download issue and getting the traceback

```
  File "/scratch/gencore/.eb/2.0/software/dram/1.3.5/bin/DRAM-setup.py", line 158, in <module>
    args.func(**args_dict)
  File "/scratch/gencore/.eb/2.0/software/dram/1.3.5/lib/python3.10/site-packages/mag_annotator/database_processing.py", line 374, in prepare_databases
    db_handler.populate_description_db(output_dbs['description_db_loc'], update_config=False)
  File "/scratch/gencore/.eb/2.0/software/dram/1.3.5/lib/python3.10/site-packages/mag_annotator/database_handler.py", line 235, in populate_description_db
    self.add_descriptions_to_database(self.make_header_dict_from_mmseqs_db(self.db_locs['uniref']) ,
  File "/scratch/gencore/.eb/2.0/software/dram/1.3.5/lib/python3.10/site-packages/mag_annotator/database_handler.py", line 155, in make_header_dict_from_mmseqs_db
    mmseqs_headers_handle = open('%s_h' % mmseqs_db, 'rb')
FileNotFoundError: [Errno 2] No such file or directory: '/scratch/Reference_Genomes/In_house/Metagenomic/oral_microbiome/dram/uniref90.20220907.mmsdb_h'
```

Here everytime i need to set the database location path as per the below command. 

```
DRAM-setup.py set_database_locations --uniref_db_loc uniref90.20220907.mmsdb
```
But this download takes more than 24 hours and on the next day, it is again complains with couldn't find  uniref90.20220908.mmsdb. We would request to fix, to update the files in version number instead of datestamps. 

I appreciate if you can fix this asap

Regards 
Jay

**Comment by rmFlynn** [comment]
I agree that this should be fixed, and I will try to do so

**Comment by rmFlynn** [comment]
Its going to be a dram 2.0 feature, it looks like.

**Comment by jayramr** [comment]
Thanks for the update

**Comment by rmFlynn** [comment]
The update is coming but as usual held up in bioconda

**Comment by jayramr** [comment]
So this is fixed in v1.4.0 ? if yes please close this ticket

**Comment by rmFlynn** [comment]
I will once 1.4 reaches bioconda
